client/src/app/elements/table/table.component.ts

```diff
@@ -418,6 +422,46 @@ export class TableComponent implements OnInit, AfterViewInit, OnDestroy {
     } else {
       let value;
 
+      if (column.resolveObservables && column.pipe) {
+        if (!column.pipe.length) {
+          column.pipe = [column.pipe as PipeType];
+        }
+
+        const pipes: any[] = [];
+        for (const [i, item] of (column.pipe as Array<PipeType>).entries()) {
+          pipes.push(
+            switchMap((data) => {
+              const result = this.ioc.columnPipe.transform(
+                data,
+                item,
+                column.pipeArguments[i],
+                rowData
+              );
+
+              const constructor = result.constructor;
+              if (constructor && (
+                constructor === Observable
+                || constructor === Subject
+                || constructor === BehaviorSubject
+                || constructor === ReplaySubject
+                || constructor === AsyncSubject
```

Comment:
This pattern might make it easier to read, what do you think?
```suggestion
[Observable, Subject, BehaviorSubject, ReplaySubject, AsyncSubject].includes(constructor)
```